src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -8,31 +8,31 @@ export interface Link {
   onClick?: () => void
 }
 
-export interface LinkElement {
-  renderItem?: ({
-    item,
-    isCurrent,
-  }: {
-    item: Link
-    isCurrent: boolean
-  }) => ReactNode
-}
+export type RenderItem = ({
+  item,
+  isCurrent,
+  className,
+}: {
+  item: Link
+  isCurrent: boolean
+  className?: string
```

Comment:
⛏️ It looks like we always pass this in?
```suggestion
  className: string
```

Reply:
@roblafeve I think this was a muscle memory thing 😂